prog/weavewait/main.go

```diff
@@ -26,6 +26,11 @@ func main() {
 		args = args[1:]
 	}
 
+	if len(args) > 0 && args[0] == "-c" { // tell pid 1 to continue
+		checkErr(syscall.Kill(1, syscall.SIGUSR2))
```

Comment:
Having looked at related questions for the issue where sigint is ignored, I do not think the Docker docs are explicit either way. 

Can you see a way to find the effective pid of a container, inside its namespace?